namer/consul/src/main/scala/io/buoyant/namer/consul/SvcAddr.scala

```diff
@@ -42,18 +42,26 @@ private[consul] object SvcAddr {
     consistency: Option[v1.ConsistencyMode] = None,
     preferServiceAddress: Option[Boolean] = None,
     tagWeights: Map[String, Double] = Map.empty,
-    stats: Stats
+    stats: Stats,
+    stateWatch: Option[PollState[http.Request, v1.IndexedServiceNodes]] = None
```

Comment:
Why is this an Option?  Isn't it always called with Some?

Reply:
I must've been overly-cautious to start with, considering it would be called from different places. Removing now